packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -232,6 +232,9 @@ const GlobalHeader = ({ className }) => {
                 <Dropdown.MenuItem href="https://learn.newrelic.com/">
                   Learn
                 </Dropdown.MenuItem>
+                <Dropdown.MenuItem href="https://observabilitypacks.gatsbyjs.io/instant-observability/">
```

Comment:
I think we may want to point this to the actual url, since it will be a hassle to get this updated again for the correct url.

Reply:
I chatted to John about this and the reason was to have a working link until we go live, i've also opened a catch-all ticket for the work to do this #450